### PR TITLE
[DOCS-63] Intro to Core

### PR DESCRIPTION
**Pull request by fonty1**
Keeping it brief, not sure what else to add here that won't be covered in the following sections.

Note, MDX formatting bugs are apparent here. This will most likely be fixed when the updated library https://github.com/streamr-dev/streamr-platform/pull/560 lands.

### CODE REVIEW

File: app/src/newdocs/content/core.mdx [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+import links from '$shared/../links'
 
 import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 
 <ScrollableAnchor id="intro-to-core"><div>
 
 ## Intro to Core
---content--
+
+<Link to={links.userpages.main}>Core</Link> is the collection of [open source](https://github.com/streamr-dev/streamr-platform), Streamr developed and supported applications that help you create, process, visualise and sell your realtime data.
```
Comment (juhah) [inline app/src/newdocs/content/core.mdx]: Seems MDX is not applied if it begins with a tag:
Screen Shot 2019-07-09 at 10 32 31

File: app/src/newdocs/content/core.mdx [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 <ScrollableAnchor id="intro-to-core"><div>
 
 ## Intro to Core
---content--
+
+<Link to={links.userpages.main}>Core</Link> is the collection of [open source](https://github.com/streamr-dev/streamr-platform), Streamr developed and supported applications that help you create, process, visualise and sell your realtime data. 
+
+Essentially, Core contains the UI toolkits for working with Streams, Canvases, Dashboards, and Products on the Streamr Platform.
+
+All you need to get started using <Link to={links.userpages.main}>Core</Link> is a <Link to={links.internalLogin}>Streamr Account</Link>.
```
Comment (juhah) [inline app/src/newdocs/content/core.mdx]: `Streamr Account` link could point to Getting started section.

File: app/src/newdocs/content/core.mdx (hunk 1 above)
Comment (juhah) [reply app/src/newdocs/content/core.mdx]: Suggestion for copy: 
> Core is the collection of open source applications developed and supported by Streamr that help you create [...]

File: app/src/newdocs/content/core.mdx [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
 import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 
 <ScrollableAnchor id="intro-to-core"><div>
 
 ## Intro to Core
---content--
+
+<Link to={links.userpages.main}>Core</Link> is the collection of [open source](https://github.com/streamr-dev/streamr-platform), Streamr developed and supported applications that help you create, process, visualise and sell your realtime data. 
+
+Essentially, Core contains the UI toolkits for working with Streams, Canvases, Dashboards, and Products on the Streamr Platform.
```
Comment (juhah) [inline app/src/newdocs/content/core.mdx]: Maybe:
> Essentially, Core is the UI toolkit for working with [...]

File: app/src/newdocs/content/core.mdx (hunk 1 above)
Comment (fonty1) [reply app/src/newdocs/content/core.mdx]: Sounds much nicer 👍

File: app/src/newdocs/content/core.mdx (hunk 2 above)
Comment (fonty1) [reply app/src/newdocs/content/core.mdx]: Yes, I was on the fence about where to point that - sure 👍

File: app/src/newdocs/content/core.mdx (hunk 1 above)
Comment (fonty1) [reply app/src/newdocs/content/core.mdx]: > Seems MDX is not applied if it begins with a tag:
> Screen Shot 2019-07-09 at 10 32 31

Yep, noted in the PR description above - it's an MDX bug that happens when the first word is a JSX link. It's likely that the new update to the MDX library will fix this.

File: app/src/newdocs/content/core.mdx (hunk 3 above)
Comment (fonty1) [reply app/src/newdocs/content/core.mdx]: Yep, this sounds better too 👍

File: app/src/newdocs/content/core.mdx (hunk 1 above)
Comment (juhah) [reply app/src/newdocs/content/core.mdx]: Or add a filler word in front, like `The <Link ...>Core</Link>`, that should fix it.

Comment (fonty1) [reply app/src/newdocs/content/core.mdx]: Fair enough, maybe Streamr Core then

**Comment by fonty1** [comment]
@juhah those fixes are in and it reads much better now. Please take another look.

**Pull request merged**